kdaglas/theStore-react
# #164530317 Fetch one product

#### What does this Pr do?
Enable users to view one of the products

#### Description of the task to be implemented?
- add the fetch one components
- add the fetchone reducer,
- create the fetchone view
- add the actions

#### How can this be manually tested?
- Fetch and check out this branch by running `git fetch origin ft-view-one-164530317 && git checkout ft-view-one-164530317`
- Install project dependencies and start the app by running `yarn install && yarn start`

#### Related Pivotal Tracker Story
[#164530317](https://www.pivotaltracker.com/story/show/164530317)
